### PR TITLE
chore: demonstrate exporting OpenTelemetry logs to Google Cloud

### PR DESCRIPTION
**Pull request by agrawal-siddharth**
Monitoring

### CODE REVIEW

**Review by yirutang — APPROVED** [review]
Please wait for Java lib owner's approval.

File: samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
+/*
```
Comment (PhongChuong) [inline samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java:1]: The general pattern for samples is to have a main file and a integration test file. E.g., see:
[SampleStorage.java](https://github.com/googleapis/java-bigquerystorage/blob/5c143a0d0e165fcacc5cf00dee5e1a391726dc5c/samples/snippets/src/main/java/com/example/bigquerystorage/StorageSample.java) and [QuickstartSampleIT.java](https://github.com/googleapis/java-bigquerystorage/blob/5c143a0d0e165fcacc5cf00dee5e1a391726dc5c/samples/snippets/src/test/java/com/example/bigquerystorage/QuickstartSampleIT.java#L50).

File: samples/install-without-bom/pom.xml [hunk 2]
```diff
@@ -22,6 +22,8 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <arrow.version>15.0.2</arrow.version>
+    <opentelemetry-exporter-logging.version>1.38.0</opentelemetry-exporter-logging.version>
```
Comment (PhongChuong) [inline samples/install-without-bom/pom.xml:25]: These can move directly into the dep as it's only referenced once.

**Review by PhongChuong — COMMENTED** [review]
Overall looks good. Just some minor suggestions.

File: samples/install-without-bom/pom.xml (hunk 2 above)
Comment (agrawal-siddharth) [reply samples/install-without-bom/pom.xml]: done

File: samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java (hunk 1 above)
Comment (agrawal-siddharth) [reply samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java:1]: Hi, I have added the files

samples/snippets/src/main/java/com/example/bigquerystorage/ExportOpenTelemetry.java
samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java

**Pull request merged**